certbot/tests/error_handler_test.py

```diff
@@ -66,9 +65,9 @@ def test_context_manager(self):
         self.init_func.assert_called_once_with(*self.init_args,
                                                **self.init_kwargs)
 
-    # On Windows, this test kills pytest itself !
-    @test_util.broken_on_windows
     def test_context_manager_with_signal(self):
+        if not self.signals:
+            self.skipTest(reason='No signals can be handled by ErrorHandler.')
```

Comment:
Since this string will sometimes be included in test output, can we modify it slightly to say:

> Signals cannot be handled on Windows.

The thinking is just making it clear to the person running the tests why there's this limitation.

Reply:
Fixed